site/docs/spec.md

```diff
@@ -473,7 +473,7 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 
 One of the interesting challenges with this is how to map Iceberg’s schema evolution (id based) on to ORC’s (name based). In theory we could use Iceberg’s column ids as the column and field names, but that would suck from a user’s point of view. 
 
-The column ids would be stored in ORC’s user metadata as “iceberg.column.id” with a comma separated list of the ids.
+The column ids would be stored in ORC’s user metadata as key `iceberg.column.mapping` with a comma separated list of the ids with format `orc_id:iceberg_id:(required|optional),orc_id:iceberg_id:(required|optional),...` per column mapping.
```

Comment:
This should be updated to use stronger language "Iceberg column IDs must be stored in ORC user metadata using the key `iceberg.column.mapping` ..."